# Description: Ignore Hugs version in TestFrown
#  ignore the version header of Hugs in the output of the test cases.
# Author: Arjan Oosting <arjanoosting@home.nl>
diff -urNad frown~/Examples/TestFrown.lhs frown/Examples/TestFrown.lhs
--- frown~/Examples/TestFrown.lhs	2006-01-13 00:09:44.000000000 +0100
+++ frown/Examples/TestFrown.lhs	2006-09-28 02:06:03.000000000 +0200
@@ -45,7 +45,7 @@
 >           opts'               =  map ("--" ++) opts
 >           (s, _)              =  revBreak (== '.') g
 >           t                   =  init s
->           diff | flag         =  "| diff -q - "
+>           diff | flag         =  "| diff -I\"|| Version: .* _\" -q - "
 >                | otherwise    =  "> "

 NB. The `|-98|' is only needed for `|LexTerm|'.
